src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -244,7 +244,7 @@ def _split_heads(self, fused_qkv):
         new_tensor_shape = fused_qkv.size()[:-1] + (self.num_heads, 3 * self.head_dim)
         # new_tensor_shape = (fused_qkv.size(1), fused_qkv.size(0)*fused_qkv.size(2), fused_qkv.size(-1))
         # fused_qkv = fused_qkv.transpose(1, 0)
-        fused_qkv = fused_qkv.reshape(*new_tensor_shape)
+        fused_qkv = fused_qkv.reshape(new_tensor_shape)
```

Comment:
Could you elaborate on why you need to remove the `*` just for learning purpose if you have time 🙏 Thanks !

Reply:
Basically, unpacking the shape means iterating over it. When tracing, this shape is actually a proxy (symbolic value), so it cannot unpack it easily. Since `view` also accepts the shape as input, an easy fix is to not unpack at all.

Reply:
Thank you very much !